developer/src/kmcmplib/src/CompMsg.cpp

```diff
@@ -150,14 +146,8 @@ const struct CompilerError CompilerErrors[] = {
     { CWARN_VirtualKeyInOutput                           , "Virtual keys are not supported in output"},
 
     { 0, nullptr }
-  };
+};
 
-KMX_CHAR *GetCompilerErrorString(KMX_DWORD code)
-{
-  for(int i = 0; CompilerErrors[i].ErrorCode; i++) {
-    if(CompilerErrors[i].ErrorCode == code) {
-      return ( KMX_CHAR*) CompilerErrors[i].Text;
-    }
-  }
-  return nullptr;
+KMX_CHAR *GetCompilerErrorString(KMX_DWORD code) {
```

Comment:
next step, make this return a `const KMX_CHAR *` ... so much `const` missing in kmcmplib.